src/Performance/PerformanceConnector.ts

```diff
@@ -0,0 +1,167 @@
+/* *
+ *
+ *  (c) 2009-2025 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Mateusz Bernacik
+ *
+ * */
+
+
+'use strict';
+
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+
+import External from '../Shared/External';
+import MorningstarAPI from '../Shared/MorningstarAPI';
+import PAUSConnector from '../Shared/PAUSConnector';
+import MorningstarURL from '../Shared/MorningstarURL';
+import PerformanceOptions, {
+    PerformanceRequestPayload
+} from './PerformanceOptions';
+
+
+/* *
+ *
+ *  Declarations
+ *
+ * */
+
+
+/* *
+ *
+ *  Functions
+ *
+ * */
+
+
+/* *
+ *
+ *  Constants
+ *
+ * */
+
+// export const DATA_TABLES = [
+//     { key: 'TrailingReturns' },
+// ];
+
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+
+export class PerformanceConnector extends PAUSConnector {
+
+
+    /* *
+     *
+     *  Constructors
+     *
+     * */
+
+
+    public constructor (
+        options: PerformanceOptions
+    ) {
+        super(
+            options
+        );
+
+        this.options = options;
+    }
+
+
+    /* *
+     *
+     *  Properties
+     *
+     * */
+
+    public override readonly options: PerformanceOptions;
+
+    /* *
+     *
+     *  Functions
+     *
+     * */
+
+
+    public override async load (
```

Comment:
Can we move most of the `load` logic from PerformanceConnector to PAUSConnector load? 
I think we discovered that the requests are rather similar/same between Hypo/Perf/XRay so maybe it would be a good spot for refactor

@pawelfus